rootfs/rootfs/usr/local/etc/init.d/docker

```diff
@@ -26,9 +26,11 @@ start() {
     if [ "$DOCKER_TLS" != "no" ]; then
         CERTHOSTNAMES="$(hostname -s),$(hostname -i)"
         for interface in ${CERT_INTERFACES}; do
-          IP=$(ip addr show ${interface} |sed -nEe 's/^[ \t]*inet[ \t]*([0-9.]+)\/.*$/\1/p')
-          if [ "$IP" != "" ]; then
-            CERTHOSTNAMES="$CERTHOSTNAMES,$IP"
+          IPS=$(ip addr show ${interface} |sed -nEe 's/^[ \t]*inet[ \t]*([0-9.]+)\/.*$/\1/p')
+          if [ "$IPS" != "" ]; then
+            for ip in ${IPS}; do
```

Comment:
Doesn't `for` handle an empty list intelligently?  ie, couldn't this whole block just be:
```bash
IPS=$(ip addr show ${interface} |sed -nEe 's/^[ \t]*inet[ \t]*([0-9.]+)\/.*$/\1/p')
for ip in $IPS; do
  CERTHOSTNAMES="$CERTHOSTNAMES,$ip"
done
```

See also:
```console
$ a=''
$ for b in $a; do echo "$b"; done
$ a='a b c'
$ for b in $a; do echo "$b"; done
a
b
c
```